tests/common/mod.rs

```diff
@@ -39,83 +39,21 @@ pub fn assert_eq_slices(expected_output: &[u8], output: &[u8], what: &str) {
 
 /// Returns a handle to a child process through which output (stdout, stderr) can
 /// be accessed.
-pub fn keepldr_exec<'a>(bin: &str, input: impl Into<Option<&'a [u8]>>) -> Output {
-    let bin_path = Path::new(CRATE).join(OUT_DIR).join(TEST_BINS_OUT).join(bin);
-
-    let mut child = Command::new(&String::from(KEEP_BIN))
+pub fn keepldr_exec<'a>(bin: impl Into<PathBuf>, input: impl Into<Option<&'a [u8]>>) -> Output {
+    let bin: PathBuf = bin.into();
+    let mut child = Command::new(KEEP_BIN)
         .current_dir(CRATE)
         .arg("exec")
-        .arg(bin_path)
+        .arg(&bin)
         .stdin(Stdio::piped())
         .stdout(Stdio::piped())
         .stderr(Stdio::piped())
         .spawn()
-        .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin, e));
-
-    if let Some(input) = input.into() {
-        child
-            .stdin
-            .as_mut()
-            .unwrap()
-            .write_all(input)
-            .expect("failed to write stdin to child");
-
-        drop(child.stdin.take());
-    }
-
-    let output = child
-        .controlled_with_output()
-        .time_limit(Duration::from_secs(TIMEOUT_SECS))
-        .terminate_for_timeout()
-        .wait()
-        .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin, e))
-        .unwrap_or_else(|| panic!("process `{}` timed out", bin));
-
-    assert!(
-        output.status.code().is_some(),
-        "process `{}` terminated by signal {:?}",
-        bin,
-        output.status.signal()
-    );
-
-    output
-}
-
-/// Returns a handle to a child process through which output (stdout, stderr) can
-/// be accessed.
-pub fn keepldr_exec_crate<'a>(
```

Comment:
Why were keepldr_exec and keepldr_exec_crate separate functions originally, and why is that no longer necessary?